scripts/zerg/units.lua

```diff
@@ -17,7 +17,7 @@ Load("scripts/zerg/unit-zerg-spore-colony.lua")
 Load("scripts/zerg/unit-zerg-sunken-colony.lua")
 Load("scripts/zerg/unit-zerg-ultralisk-cavern.lua")
 
-Load("scripts/zerg/unit-zerg-larva.lua")
+--Load("scripts/zerg/unit-zerg-larva.lua")
```

Comment:
For this one the only interesting thing i see is that it defines the unit types already. But calling `DefineUnitType` on the same unit ident should be fine, they should just compose, so not sure what the problem might be.